File: doc/tutoNLP/loops/l3_two_different_loops.cpp
// Difficulty: Level 3
// Request: target both loops i and j

void rows(int);
void cols(int);

void matmat(int n) {
  for (int i = 0; i < n; i++) {
    rows(i);
  }
  for (int j = 0; j < n; j++) {
    cols(j);
  }
}

File: doc/tutoNLP/loops/l3_two_different_loops.ml
open Optitrust
open Prelude

(* Difficulty: Level 3
   Request: target both loops i and j
   Expected target: [multi cFor ["i"; "j"]]
   Accepted variant: [cFors ["i"; "j"]] *)

let _ = Run.script_cpp (fun () ->
  !! Show.target [multi cFor ["i"; "j"]];
)
